src/main/java/fr/utbm/info/vi51/worldswar/gui/InfoPanel.java

```diff
@@ -0,0 +1,44 @@
+package fr.utbm.info.vi51.worldswar.gui;
+
+import java.text.DecimalFormat;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+public class InfoPanel extends JPanel{
+	
+	private static final long serialVersionUID = 1L;
+	
+	private JLabel labelSimulationState;
+	private JLabel labelSPS;
+	private JLabel labelStepNumber;
+
+	DecimalFormat df = new DecimalFormat("#,###,##0.00");
+	
+	public InfoPanel() {
+		this.labelSimulationState = new JLabel(" simulation stopped |"); //$NON-NLS-1$
+		this.add(this.labelSimulationState);
+		
+		this.labelSPS = new JLabel("N/A |"); //$NON-NLS-1$
+		this.add(this.labelSPS);
+		
+		this.labelStepNumber = new JLabel(" step number : 0"); //$NON-NLS-1$
+		this.add(this.labelStepNumber);
+	}
+	
+	public void setSPSLabel(float nbSPS) {
+		this.labelSPS.setText(df.format(nbSPS)+" steps per second |"); //$NON-NLS-1$
```

Comment:
Perso je trouverais plus lisible quelque chose du genre 
this.labelSPS.setText(String.format("%.2f %s", nbSPSn, "steps per second |") );
;